backend/editor/entries.py

```diff
@@ -53,7 +75,7 @@ def update_nodes(label, entry, incomingData):
             raise ValueError("Invalid key: %s", key)
     
     # Build query
-    query = [f"""MATCH (n:{label}) WHERE n.id = $id"""]
+    query = [f"""MATCH (n:{label}) WHERE n.id = $id """, """SET n={} """, """SET n.id = $id"""]

```

Comment:
The idea here is that your reset the node before running updates ?

Reply:
Yess. The React App sends all the current keys of a node and the backend updates only those keys. But, if a user deletes any key, the change doesn't reflect. Hence, I clear the node and start adding all the keys sent by the React App.

Reply:
If there's a better way to approach this, do let me know.

Reply:
Hi @aadarsh-ram, the way to remove a property could be to just set it to `null`

So if I want to remove the "tags_fr" key, I send "tags_fr: null".